File: models/inquiry_validation.go
package models

type InquiryValidation struct {
	ID                 int  `json:"id" yaml:"id"`
	ServiceId          int  `json:"service_id" yaml:"serviceId" gorm:"default:1"`
	IsRequiredSubject  bool `json:"is_required_subject" yaml:"isRequiredSubject" gorm:"default:1"`
	IsRequiredEmail    bool `json:"is_required_email" yaml:"isRequiredEmail" gorm:"default:1"`
	IsRequiredCategory bool `json:"is_required_category" yaml:"isRequiredCategory" gorm:"default:1"`
	IsRequiredText     bool `json:"is_required_text" yaml:"isRequiredText" gorm:"default:1"`
}

File: cpanel/src/models/inquiry-validation.ts
export interface InquiryValidation {
  id: number;
  service_id: number;
  is_required_subject: boolean;
  is_required_email: boolean;
  is_required_category: boolean;
  is_required_text: boolean;
};

export const initInquiryValidation = () => {
  return {
    id: 0,
    service_id: 0,
    is_required_subject: true,
    is_required_email: true,
    is_required_category: true,
    is_required_text: true
  };
};
